ovirtlago/reposetup.py

```diff
@@ -72,33 +73,55 @@ def wrapper(*args, **kwargs):
 def _fix_reposync_issues(reposync_out, repo_path):
     """
     Fix for the issue described at::
-        https://bugzilla.redhat.com/show_bug.cgi?id=1332441
+        https://bugzilla.redhat.com//show_bug.cgi?id=1399235
+        https://bugzilla.redhat.com//show_bug.cgi?id=1332441
+
     """
-    LOGGER.warn(
-        'Due to bug https://bugzilla.redhat.com/show_bug.cgi?id=1332441 '
-        'sometimes reposync fails to update some packages that have older '
-        'versions already downloaded, will remove those if any and retry'
+    if any([len(repo_path) == 0, len(reposync_out) == 0]):
+        LOGGER.warning(
+            (
+                'unable to run _fix_reposync_issues, no reposync output '
+                ' or empty repo path.'
+            )
+        )
+        return
+    rpm_regex = r'[a-z]{1}[a-zA-Z0-9._\\-]+'
+    wrong_version = re.compile(
+        r'(?P<package_name>' + rpm_regex + '): \[Errno 256\]'
+    )
+    wrong_release = re.compile(r'(?P<package_name>' + rpm_regex + ') FAILED')
+    packages = set(
+        itertools.chain(
+            wrong_version.findall(reposync_out),
+            wrong_release.findall(reposync_out)
+        )
+    )
+    count = 0
+    LOGGER.debug(
```

Comment:
As per the logging - I don't think it matters, because eventually the user will see packages were removed in the output(in stdout, not only in the the log file), and this is a reposync bug, so there is nothing he can really debug here.
About the regex split few lines above(one for each bug) - I think that will help us debug/improve the code itself it in the future, if we encounter new issues with reposync.